apps/language_server/test/providers/execute_command/manipulate_pipes/ast_test.exs

```diff
@@ -0,0 +1,159 @@
+defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes.ASTTest do
+  use ExUnit.Case
+
+  alias ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes.AST
+
+  describe "to_pipe/1" do
+    test "single-line selection with two args in named function" do
+      assert "A.B.C.a |> X.Y.Z.function_name(b)" == AST.to_pipe("X.Y.Z.function_name(A.B.C.a, b)")
```

Comment:
```suggestion
      assert AST.to_pipe("X.Y.Z.function_name(A.B.C.a, b)") == "A.B.C.a |> X.Y.Z.function_name(b)"
```

For these tests can we put the expected result on the right side? Roughly we follow https://github.com/christopheradams/elixir_style_guide#testing